composer.json

```diff
@@ -1,16 +1,22 @@
 {
   "name": "spryker-eco/econda",
+  "type": "library",
   "description": "Econda module",
-  "license": "MIT",
+  "license": "proprietary",
```

Comment:
This doesnt fit the other files and licence doc block now.

Reply:
Reverted.